src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,21 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.FigureSupplier
+
+private const val NUMBER_OF_FIGURE = 6
+private const val DIVIDER = 2
+
 fun main() {
-    // draw all figures here
+    val figureSupplier = FigureSupplier()
+    val figures = arrayOfNulls<Figure>(NUMBER_OF_FIGURE)
+    for (i in 0 until figures.size / DIVIDER) {
```

Comment:
Please try to refactor using forEachIndexed() to avoid code duplication
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each-indexed.html